core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/validation/issue/ValidationIssue.java

```diff
@@ -52,6 +58,8 @@
 	private final short affectedComponentType;
 	private final boolean whitelisted;
 	
+	private Map<String, Object> details = newHashMap();
```

Comment:
Use `Collections.emptyMap()` or `null` here so fetching issues without details does not create a bunch of empty maps.